### DISCUSSION TITLE
Setup self-hostname renovate with Bitbucket Data Center failed

### DISCUSSION THREAD

**Discussion by shenxianpeng** [renovatebot/renovate#22370] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

35.98.4

### If you're self-hosting Renovate, select which platform you are using.

Bitbucket Server

### If you're self-hosting Renovate, tell us what version of the platform you run.

atlassian/bitbucket-server:5.12.3

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

I'm trying to setup self-hostname renovate and follow the [Bitbucket Data Center and Server documentation ](https://docs.renovatebot.com/modules/platform/bitbucket-server/)step by step but failed when I ran the following commands.

It looks like yarn start should be yarn run, because I did not see the start command from https://yarnpkg.com/en/docs/cli/run

Not sure if I missed some prerequisite configurations that the above documentation not mentioned, I appreciate any suggestions.

## Error is below

```bash
$ yarn start --platform=bitbucket-server --endpoint=http://localhost:7990 --git-fs=http --username=renovate-bot --password=abc123456789! --log-level=debug --autodiscover=true
yarn run v1.22.19
error Command "start" not found.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

Tried to change `start` to `run` and run again

```bash
$ yarn run --platform=bitbucket-server --endpoint=http://localhost:7990 --git-fs=http --username=renovate-bot --password=abc123456789! --log-level=debug --autodiscover=true
yarn run v1.22.19
error Command "--endpoint=http://localhost:7990" not found.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```bash

### What you tried so far.

Also tried the Alternatively using env, but the error the same

```bash
export RENOVATE_PLATFORM=bitbucket-server
export RENOVATE_ENDPOINT=http://localhost:7990
export RENOVATE_GIT_FS=http
export RENOVATE_USERNAME=renovate-bot
export RENOVATE_PASSWORD=abc123456789!
export LOG_LEVEL=debug
```

```
$ yarn start --autodiscover=true
yarn run v1.22.19
error Command "start" not found.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

# Change to run, and execute again
$ yarn run --autodiscover=true
yarn run v1.22.19
info Commands available from binary scripts: acorn, ejs, escodegen, esgenerate, esparse, esvalidate, hexo, is-docker, jake, js-yaml, marked, mime, node-which, nunjucks-precompile, resolve, stylus, to-title-case
info Project commands
   - build
      hexo generate
   - clean
      hexo clean
   - deploy
      hexo deploy
   - server
      hexo server
question Which command would you like to run?: clean
$ hexo clean
INFO  Validating config
INFO  Deleted database.
INFO  Deleted public folder.
Done in 11.96s.
```

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by rarkins** [comment, marked as answer]
Those instructions are for running Renovate _from source_ (i.e. a clone of this repo).

How did you download/install Renovate? e.g. from npm, or a Docker Hub image?

**Reply by shenxianpeng** [reply]
Tried under the source of Renovate it works. Thank you @rarkins
